vuepress/docs/next/tutorials/getting-started/openshift-install.md

```diff
@@ -5,117 +5,90 @@ sidebarDepth: 2
+oc apply -n entando -f https://raw.githubusercontent.com/entando/entando-releases/v7.0.0/dist/ge-1-1-6/namespace-scoped-deployment/namespace-resources.yaml
 ```
 
-### Get your Cluster Default Ingress
-
-If you're deploying on a managed cluster, get the default hostname from your cluster administrator. Entando uses wildcard addressing to connect different parts of your Entando Application, which requires the default route for applications exposed on your cluster. You'll set this value in step 3 below.
-
-### Setup and Deploy
-1. Select, download and unpack the entando-helm-quickstart release:
-<https://github.com/entando-k8s/entando-helm-quickstart/releases>
+2. (Optional) A ConfigMap can be used to modify the behavior of the Entando Operator. Refer to the [Entando Operator](../../tutorials/devops/entando-operator.md) page for more information.
 
- - See the included README file for more information on the following steps.
-
- ```
- curl -sfL https://github.com/entando-k8s/entando-helm-quickstart/archive/v6.3.2.tar.gz | tar xvz
- ```
-
-2. Navigate to the new directory
-```
-cd entando-helm-quickstart-6.3.2
-```
-3. Edit `sample-configmaps/entando-operator-config.yaml`
-   - If you're deploying to a managed cluster:
-      - Set `entando.default.routing.suffix` to the default URL of applications deployed in your OpenShift cluster. If you're unsure of this value, please check with your cluster administrator.
-      - Entando will create applications using the default URL and relies on wildcard DNS resolution.
-   - If you're using CRC:
-      - Set `entando.default.routing.suffix` to the value from `crc ip`, plus `nip.io`. For example, `entando.default.routing.suffix: 192.168.64.10.nip.io`.
-
-4. Deploy your ConfigMap
-```
-oc apply -n entando -f sample-configmaps/entando-operator-config.yaml
+## Configure the Entando Application
+1. Download the `entando-app.yaml` template
+```sh
+curl -sLO "https://raw.githubusercontent.com/entando/entando-releases/v7.0.0/dist/ge-1-1-6/samples/entando-app.yaml"
 ```
 
-5. Update Helm dependencies
-```
-helm dependency update
-```
-6. Run Helm to generate the template file
-```
-helm template my-app --namespace=entando ./ > my-app.yaml
-```
-   - If you're using Helm 2 instead of Helm 3, then replace ```helm template my-app``` with ```helm template --name=my-app```.
-7. Deploy Entando
-```
-oc create -f my-app.yaml
+2. Determine the hostname for your application, YOUR-HOST-NAME.
+  - If you're deploying to a managed cluster:
+    - Determine the default host name of your cluster. If you're unsure of the default host name, please check with your cluster administrator.
+    - Add a prefix with the name of your project or application.  
+    - For example, YOUR-HOST-NAME could be `my-app.apps.cluster-4bb2.4bb2.example.opentlc.com` where `apps.cluster-4bb2.4bb2.example.opentlc.com` is the host name of the OpenShift cluster.
```

Comment:
`my-app.apps.cluster-4bb2.4bb2.example...>> `your-app.apps.cluster-4bb2.4bb2.example

Reply:
I this case my-app is the default appname that the operator provides so I'm going to leave it as my. :(